BitwardenShared/Core/Platform/Repositories/SettingsRepositoryTests.swift

```diff
@@ -186,25 +186,28 @@ class SettingsRepositoryTests: BitwardenTestCase {
         }
     }
 
-    /// `foldersListPublisher()` returns a decrypted flow of the user's folders.
+    /// `foldersListPublisher()` returns a decrypted flow of the user's folders in sorted order.
```

Comment:
⛏️ Just one minor enhancement would be to say sorted by what order, in this case alphabetically ascending.
```suggestion
    /// `foldersListPublisher()` returns a decrypted flow of the user's folders in ascending alphabetically order.
```